src/js/components/login/LoginMax.jsx

```diff
@@ -0,0 +1,61 @@
+/**
+  * LoginMax.jsx
+  * Createdd by Kevin Li 10/13/16
+  **/
+
+import React from 'react';
+import _ from 'lodash';
+import Cookies from 'js-cookie';
+
+import { kGlobalConstants } from '../../GlobalConstants.js';
+import ErrorMessage from '../SharedComponents/ErrorMessage.jsx';
+
+export default class LoginMax extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            redirect: ''
+        };
+    }
+
+    componentDidMount() {
+        this.detectRedirection();
+    }
+
+    componentDidUpdate(prevProps, prevState) {
+        if (!_.isEqual(prevProps.location, this.props.location)) {
+            this.detectRedirection();
+        }
+    }
+
+    detectRedirection() {
+        // check if the URL has a redirect param, save it in the state
+        if (this.props.location.query.hasOwnProperty('redirect')) {
+            this.setState({
+                redirect: this.props.location.query.redirect
+            }, () => {
+                // save the redirect destination as a cookie, expire after 5 min
+                Cookies.set('brokerRedirect', this.state.redirect, {expires: (5/(24*60))});
+            });
+        }
+        else {
+            this.setState({
+                redirect: ''
+            }, () => {
+                // remove the redirect destination cookie
+                Cookies.remove('brokerRedirect');
+            });
+        }
+    }
+
+	render() {
+		return (
+                <div className="row">
+                    <div className="col-xs-12">
+                    	<p className="instructions">Sign in or register for the DATA Act Broker using your MAX ID.</p>
+                        <a href={kGlobalConstants.CAS_ROOT + '/cas/login?service=' + encodeURIComponent(kGlobalConstants.AUTH_CALLBACK)} className="usa-da-button btn-primary btn-lg btn-full">Sign In Using MAX</a>
```

Comment:
Local installs use the old login approach https://github.com/fedspendingtransparency/data-act-broker-web-app/blob/feature-maxsso/src/js/containers/login/LoginContainer.jsx#L53